### PR TITLE
fix: use the correct name for survey

### PR DESCRIPTION
**Pull request by Steffy29**
Ref: UXCT-136

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -785,11 +785,6 @@
     rollup-pluginutils "^2.3.3"
     slash "^2.0.0"
 
-"@ovh-ux/ng-translate-async-loader@^2.0.0":
```
Comment (FredericEspiau) [inline yarn.lock]: Why?

File: src/resiliation/resiliation-reason.component.js [hunk 2]
```diff
@@ -49,7 +49,7 @@ export default {
       self.panel = false;
       self.onChange({
         ELEMENT: self.tucResiliationReason,
-        SURVEY: self.tucResiliationReasonModel,
```
Comment (FredericEspiau) [inline src/resiliation/resiliation-reason.component.js]: Good find

Comment (Steffy29) [reply src/resiliation/resiliation-reason.component.js]: No it wasn't set with any value.

File: yarn.lock (hunk 1 above)
Comment (Steffy29) [reply yarn.lock]: I don't know, it was updated by itself.

Comment (FredericEspiau) [reply yarn.lock]: Can you discard it then please ? :)

File: src/resiliation/resiliation-reason.component.js (hunk 2 above)
Comment (FredericEspiau) [reply src/resiliation/resiliation-reason.component.js]: It was a compliment :)

File: src/resiliation/resiliation-reason.component.js [hunk 3]
```diff
@@ -49,7 +49,7 @@ export default {
...
+        SURVEY: self.resiliationReasonModel,
```
Comment (cbourgois) [inline src/resiliation/resiliation-reason.component.js]: I think there is a problem here : https://github.com/ovh-ux/ovh-manager-telecom/blob/213b4783df/client/app/telecom/pack/xdsl/resiliation/pack-xdsl-resiliation.html#L70-L75

It should be 

```html 
<tuc-resiliation-reason
    data-ng-show="PackXdslResiliation.resiliationTerms.status"
    data-tuc-resiliation-reason-filter="{{PackXdslResiliation.resiliationTerms.data.resiliationReasons.join(',')}}"
    data-tuc-resiliation-reason="{id: PackXdslResiliation.resiliationTerms.serviceName, name:PackXdslResiliation.resiliationTerms.serviceName, type:'service', typeName:PackXdslResiliation.resiliationTerms.typeName}"
    on-change="PackXdslResiliation.resiliateService(ELEMENT, SURVEY, ACCEPT)">
</tuc-resiliation-reason>
```

(wrong `data-tuc-resiliation-reason-filter` binding).

We have another problem in this same code, there is no `tucResiliationReasonModel` defined https://github.com/ovh-ux/ng-ovh-telecom-universe-components/blob/f09253603bd2cb81d2219c8eecf57cace0477cdb/src/resiliation/resiliation-reason.component.js#L9-L14

so the `canResiliate` method will always return `false` 
https://github.com/ovh-ux/ng-ovh-telecom-universe-components/blob/f09253603bd2cb81d2219c8eecf57cace0477cdb/src/resiliation/resiliation-reason.component.js#L38-L46

Comment (FredericEspiau) [reply src/resiliation/resiliation-reason.component.js]: Even better find :D

File: src/resiliation/resiliation-reason.component.js (hunk 2 above)
Comment (Steffy29) [reply src/resiliation/resiliation-reason.component.js]: Sorry I didn't understand due to my english level ;-)

File: src/resiliation/resiliation-reason.component.js (hunk 3 above)
Comment (Steffy29) [reply src/resiliation/resiliation-reason.component.js]: Thanks @FredericEspiau !

File: yarn.lock (hunk 1 above)
Comment (Steffy29) [reply yarn.lock]: Did you remove it?

File: src/resiliation/resiliation-reason.component.js (hunk 3 above)
Comment (Steffy29) [reply src/resiliation/resiliation-reason.component.js]: @cbourgois no it won't, I made some tests on it.

Comment (Steffy29) [reply src/resiliation/resiliation-reason.component.js]: @cbourgois I made the fix about data-tuc-resiliation-reason on telecom manager

Comment (antleblanc) [reply src/resiliation/resiliation-reason.component.js]: @Steffy29 Shouldn't be `tucResiliationReasonModel` instead?

Comment (Steffy29) [reply src/resiliation/resiliation-reason.component.js]: No I made some tests yesterday with the older version with `self.tucResiliationReasonModel` and the `SURVEY` wasn't set.

**Pull request merged**